scripts/start_civ.sh

```diff
@@ -558,6 +561,20 @@ function setup_qmp_pipe() {
     GUEST_QMP_PIPE="-qmp pipe:$qmp_pipe"
 }
 
+function set_guest_pwr_vol_button() {
+    echo "Guest graphic mode is $GUEST_GRAPHIC_MODE"
+    if [[ $GUEST_GRAPHIC_MODE == "GVT-d" ]]; then
```

Comment:
I think no need to check graphic mode, even graphic mode is not GVT-d, we can also have this feature, right?